docs/notebooks/test_snia.ipynb

```diff
@@ -42,30 +57,48 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c87c4488-9368-4a0a-8400-1596bb15492a",
-   "metadata": {},
+   "id": "3742a30a04a1b62c",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-26T15:38:20.797258Z",
+     "start_time": "2024-09-26T15:38:20.672792Z"
+    }
+   },
    "outputs": [],
    "source": [
-    "from test_snia import test_snia_end2end"
+    "import conftest\n",
+    "from sources.test_snia import test_snia_end2end"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "3285d471-c454-4c7a-b7cb-b8a79ab7ec98",
-   "metadata": {},
+   "id": "58a2bc03f3a3a3a0",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-26T15:38:47.963561Z",
+     "start_time": "2024-09-26T15:38:20.802799Z"
+    }
+   },
    "outputs": [],
    "source": [
-    "res = test_snia_end2end(\n",
-    "    None, opsim_db_file=None, opsim=False, nsample=10, return_result=True, phase_rest=np.linspace(-15, 45, 20)\n",
+    "res, passbands = test_snia_end2end(\n",
```

Comment:
My preference would be not to use `test_snia_end2end` here and instead repeat the code in the notebook. I know code repetition is usually considered bad, but I think the code of interest to the notebook user will be what is in that function. Leaving the code out of the notebook doesn't provide the user all the information.

Alternatively, if we really want to keep code in sync, we could break out modules into new files.

Reply:
We have planned to update notebook to show how model is defined with more text descriptions by breaking the end2end test into multiple steps. Right now I prefer to keep it this way so I don't need to change the same thing in two different places when experimenting.

Reply:
I agree with this, but I've just keep the same NB structure as it was before. I'd keep these changes out of the scope of this PR or ask someone to help me commiting/PRing these changes